src/collapse/collapse.stories.tsx

```diff
@@ -0,0 +1,177 @@
+import React, {useState} from 'react';
+
+import ChevronDownIcon from '@jetbrains/icons/chevron-down';
+
+import ChevronUpIcon from '@jetbrains/icons/chevron-up';
+
+import Button from '../button/button';
+import Icon from '../icon/icon';
+
+import LoaderInline from '../loader-inline/loader-inline';
+
+import Collapse from './collapse';
+import CollapseContent from './collapse-content';
+import CollapseControl from './collapse-control';
+
+import styles from './collapse.stories.css';
+
+export default {
+  title: 'Components/Collapse',
+
+  parameters: {
+    notes: 'The "Collapse" component hides content and shows more when clicked, saving space and keeping it tidy.',
+
+    hermione: {
+      actions: [
+        {type: 'click', selector: '[data-test~=ring-collapse-control]'}
+      ]
+    }
+  }
+};
+const text = `This is very long text! This is very long text! This is very long text! This is very
+            long text! This is very long text! This is very long text! This is very long text! This
+            is very long text! This is very long text! This is very long text! This is very long
+            text! This is very long text! This is very long text! This is very long text! This is
+            very long text! This is very long text! This is very long text! This is very long text!
+            This is very long text! This is very long text! This is very long text!`;
+
+export const Basic = () => (
```

Comment:
There is no screenshot gathered for this components, would you capture them?

Reply:
@andrey-skl I'll add them, thank you 👍